i18n/id/white-paper.md

```diff
@@ -0,0 +1,361 @@
+
+Kedua aplikasi token tea melayani fungsi vital dalam mendukung dan menumbuhkan ekosistem sumber terbuka. Mempertaruhkan tea akan memastikan bahwa sistem tea terus beroperasi dengan aman, sehingga semua peserta jaringan dapat mengirimkan dan mengakses paket untuk memeriksanya, mengintegrasikannya ke dalam aplikasi mereka, dll. Sebaliknya, seduhan tea akan mendukung tujuan tea untuk menyediakan alat bagi semua peserta jaringan untuk mendukung dan menggunakan paket yang memenuhi persyaratan kualitas dan ketergantungan, seperti yang dirumuskan oleh komunitas tea melalui dukungan dan perbedaan pendapat mereka terhadap setiap paket. Kehati-hatian akan dilakukan ketika mendefinisikan dan mengimplementasikan parameter mempertaruhkan dan menyeduh, sehingga yang satu tidak menjadi parasit bagi yang lain.
+
+## Insentif dan Hukuman
+
+Seperti yang telah dibahas sebelumnya, bisa jadi ada insentif yang kuat bagi pelaku kejahatan untuk mengkompromikan perangkat lunak sumber terbuka. Mayoritas infrastruktur penting Internet berjalan pada sumber terbuka, dan perlombaan untuk menemukan eksploitasi dan kerentanan lainnya sedang berlangsung. Di tea, kami percaya bahwa manajer/pengelola paket bukanlah orang yang harus disalahkan (meskipun mereka sering disalahkan).
+
+Insentif protokol tea memperbaiki hal ini melalui distribusi insentif yang adil dan merata. Paket seperti Lodash dengan lebih dari 151 ribu dependen adalah pilar pengembangan sumber terbuka, dan pengelolanya layak diberi penghargaan secara proporsional. Namun, sistem penghargaan yang dibangun semata-mata pada jumlah tanggungan akan mencegah inovator mengganggu monopoli ini kecuali mereka cukup didanai oleh pihak ketiga atau telah mengumpulkan sumber daya yang cukup untuk mendanai diri sendiri. Pendekatan ini kemungkinan besar akan menyebabkan menyusutnya jumlah kontributor, sehingga menghasilkan kebalikan dari apa yang dimaksud dengan tea.
+
+Tujuan tea adalah untuk mewakili nilai perangkat lunak sumber terbuka dan, dengan demikian, mendorong pertumbuhannya dengan memberdayakan para pesertanya dengan sumber daya yang mereka butuhkan untuk mengejar hasrat mereka tanpa terbebani. Sistem distribusi insentif tea perlu mempertimbangkan dengan cermat rasio seduhan setiap paket dan menyesuaikan insentif setiap paket dengan tepat. Untuk mengurangi risiko sejumlah kecil paket yang digunakan sebagai dependensi di banyak aplikasi yang mengumpulkan sebagian besar reward menyeduh, kami akan memanfaatkan penelitian yang dihasilkan oleh web3 Foundation[^ 19] untuk mekanisme reward berbasis bukti kepemilikan Polkadot. Kita bisa menyesuaikan lebih lanjut implementasi dan variabel-variabelnya berdasarkan hasil eksperimen praktis.
```

Comment:
> [^ 19]

There should be no space between the characters --> [^19]

Reply:
Please double-check your references. According to the white-paper.md file, this should be reference `[^22]`